src/thing/add.test.ts

```diff
@@ -607,6 +608,173 @@ describe("addDatetime", () => {
   });
 });
 
+describe("addDate", () => {
+  it("adds the given datetime value for the given predicate", () => {
+    const thing = mockThingFrom("https://some.pod/resource#subject");
+
+    const updatedThing = addDate(
+      thing,
+      "https://some.vocab/predicate",
+      new Date(Date.UTC(1990, 10, 12))
+    );
+
+    expect(
+      updatedThing.predicates["https://some.vocab/predicate"].literals![
+        "http://www.w3.org/2001/XMLSchema#date"
+      ]
+    ).toStrictEqual(["1990-11-12T00:00:00.000Z"]);
```

Comment:
I think this is the result of the `serializeDate` function just calling `toISOString()`, but I think this should be:

```suggestion
    ).toStrictEqual(["1990-11-12Z"]);
```